### DISCUSSION TITLE
Binding classes only works if the class is .hidden

### DISCUSSION THREAD

**Discussion by SimonPadbury** [alpinejs/alpine#1851] [7. Bugs (archived)] [answered]
Hi, the examples in the docs work OK at https://alpinejs.dev/directives/bind -- but if I use any class names other than 'hidden', it doesn't work. This for Alpine 3.2.2

**Comment by SimoTod** [comment, marked as answer]
It seems to work correctly (I would be surprised if it didn't, people would have picked that up already): https://codepen.io/SimoTod/pen/bGWaMEp

There's probably something missing in your code. Can you post an example that does not work?

**Comment by SimonPadbury** [comment]
Hi Simone,

Thankyou — it is working OK. False alarm. Sorry, my internet was slow and Alpine wasn’t coming from the CDN.
